src/App.jsx

```diff
@@ -1,5 +1,37 @@
+import { useCookies } from 'react-cookie';
+import { useState, useEffect } from 'react';
+import ApiService from './services/ForecastApiService';
+
 function App() {
-  return <div>Hello world!</div>;
+  const [cookies] = useCookies(['token']);
+  const [locations, setLocations] = useState([]);
+
+  useEffect(async () => {
+    const searchedLocations = await getSearchLocation('London', cookies);
+    setLocations(searchedLocations.locations);
+  }, []);
+
+  async function getSearchLocation(location, cookies) {
+    const accessToken = await ApiService.getAccessToken(cookies);
```

Comment:
what about excception handling? try/catch?

Reply:
Done. I added try/catch to methods for handling exceptions.